### PR TITLE
Added up to Tuple10.

### PR DESCRIPTION
**Pull request by vinjana**
My biggest question concerns Tuple13 and the field naming.

### CODE REVIEW

File: RoddyCore/src/de/dkfz/roddy/knowledge/files/Tuple13.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
         M extends FileObject
         > extends AbstractFileObjectTuple {
 
+    // TODO: @dankwart-de: Why has Tuple13 different names for the fields then e.g. Tuple2??
```
Comment (dankwart-de) [inline RoddyCore/src/de/dkfz/roddy/knowledge/files/Tuple13.java]: For that, you need to know, how I named the tuple fields in the first tuple implementations. I took the names from computer graphics vectors, namely x, y, z and w / r (or else). As you can see above, this mechanism gets confusing and misleading for higher order vectors and stops working at some point. That's why I started the tuple13 with _a

File: RoddyCore/src/de/dkfz/roddy/knowledge/files/Tuple8.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2016 eilslabs.
```
Comment (dankwart-de) [inline RoddyCore/src/de/dkfz/roddy/knowledge/files/Tuple8.java]: It is not eilslabs anymore. It is "The Roddy WMS developers" or DKFZ or so.

File: RoddyCore/src/de/dkfz/roddy/knowledge/files/Tuple9.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        R extends FileObject,
+        Q extends FileObject,
+        W extends FileObject,
+        S extends FileObject,
+        T extends FileObject,
+        U extends FileObject>
+        extends Tuple8<X, Y, Z, R, Q, W, S, T> {
+    private final U value8;
+
+    public Tuple9(X value0, Y value1, Z value2, R value3, Q value4, W value5, S value6, T value7, U value8) {
```
Comment (dankwart-de) [inline RoddyCore/src/de/dkfz/roddy/knowledge/files/Tuple9.java]: As these are all new objects, I'd vote for a better naming scheme (like in tuple13). Maybe go from _a to ... _z / a ... z. Here the order is confusing.

File: RoddyCore/src/de/dkfz/roddy/knowledge/files/Tuple7.java [hunk 4]
```diff
@@ -9,17 +9,19 @@
 /**
  * A file object tuple designed for (generic) job calls with 5 entries
```
Comment (vinjana) [inline RoddyCore/src/de/dkfz/roddy/knowledge/files/Tuple7.java]: fix comment

**Pull request merged**